docs/src/main/paradox/journal.md

```diff
@@ -63,7 +63,7 @@ a single `persistence_id`.
 in each Cassandra partition. It is a target as `persistAll` calls will have all the events in the same partition
 even if it will exceed the target partition size to ensure atomicity.
 
-It is not possible to change the value once you have data so consider if the default of 50000 is right for your
+It is not possible to change the value once you have data so consider if the default of 500000 is right for your
```

Comment:
For reference: https://github.com/akka/akka-persistence-cassandra/blob/master/core/src/main/resources/reference.conf#L133